syntax = "proto3";

package grpc;

message Empty {}

message Message {
    string name = 1;
    string message = 2;
}

message ChatStreamAuth {
    string name = 1;
}

service ChatServer {
    rpc ChatStream (ChatStreamAuth) returns (stream Message);
    rpc SendMessage (Message) returns (Empty);
}
